assets/js/screens/Collection/Collection.jsx

```diff
@@ -24,6 +25,17 @@ const ScreensCollection = () => {
   const { data, loading, error } = useQuery(GET_COLLECTION, {
     variables: { id },
   });
+  const [updateCollection] = useMutation(UPDATE_COLLECTION, {
+    onCompleted({ updateCollection }) {
+      toastWrapper(
+        "is-success",
+        `Collection has been ${
+          updateCollection.published ? "published" : "unpublished"
+        }`
+      );
+    },
+  });
```

Comment:
In advance of @kdid 's issue about handling errors, can we add an `onError()` handler here in case something fails?